src/applications/accredited-representative-portal/accreditation/21a/containers/Form.jsx

```diff
@@ -2,20 +2,19 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
-
 import formConfig from '../config/form';
 
-const App = ({ location, children }) => {
+function Form({ location, children }) {
```

Comment:
any reason for `function` vs `const`?

Reply:
I really don't have a rationale, nor do I think it's so easy to glean a pattern repo-wide or app-wide. However in some areas of the codebase I've seen higher level functions as `function` and smaller functions as `const` arrow fn. I don't love the rationale and in projects where I have a say on the predominant patterns, I'd rather base one on the ramifications of the functional differences between the two (e.g. will the difference in `this` binding ever matter much when writing functional React) and apply that consistently.